docs/src/index.md

```diff
@@ -18,6 +18,84 @@ The Enzyme binary dependencies will be installed automatically via Julia's binar
 
 The Enzyme.jl API revolves around the function [`autodiff`](@ref), see it's documentation for details and a usage example. Also see [Implementing pullbacks](@ref) on how to use Enzyme.jl to implement back-propagation for functions with non-scalar results.
 
+## Getting started
+
+```julia
+rosenbrock(x, y) = (1.0 - x)^2 + 100.0 * (y - x^2)^2
+rosenbrock_inp(x) = 1.0 - x[1])^2 + 100.0 * (x[2] - x[1]^2)^2
```

Comment:
missing `(`. Should be `rosenbrock_inp(x) = (1.0 - x[1])^2 + 100.0 * (x[2] - x[1]^2)^2`

Reply:
Good catch